### PR TITLE
Bump to 2.0.1.

### PR DESCRIPTION
**Pull request by jsha**
Brings in a fix to 308 redirects.

### CODE REVIEW

**Review by algesten — APPROVED** [review]
Go for it.

**Comment by jsha** [comment]
I'm actually going to hold this release until #291 lands so we can get both in one go.

**Comment by jyn514** [comment]
https://github.com/algesten/ureq/pull/291 was merged :)

**Pull request merged**
